update setup

hipraveenp opened the pull request:
- Added validation check for existing installations.
- Added support for Postgresql database

hipraveenp committed:
update setup

sibidharan commented:
I see the code limits the installation to PHP 8.3, but OpenSwoole's minimum is 7.4, and I set the same in composer.json. So lets check if its minimum 7.4, or better read the minimum from composer.json and validate the same. If possible, run a test on PHP 7.4 and see if the basic functionalities are working.

hipraveenp committed:
Merge branch 'sibidharan:master' into master

hipraveenp committed:
Update in installation steps

- Fixed OpenSwoole ini file
- Added condition for the required minimum PHP version

hipraveenp commented:
> I see the code limits the installation to PHP 8.3, but OpenSwoole's minimum is 7.4, and I set the same in composer.json. So lets check if its minimum 7.4, or better read the minimum from composer.json and validate the same. If possible, run a test on PHP 7.4 and see if the basic functionalities are working.

I will test the functionalities later, for now added the condition for PHP's minimum version of 7.4

sibidharan committed:
Merge branch 'master' into master
